### PR TITLE
[MIG] 15.0 mail_environment

### PR DESCRIPTION
**Pull request by JuMiSanAr**
Migrate mail_environment to version 15.

### CODE REVIEW

File: oca_dependencies.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+server-env https://github.com/camptocamp/server-env 15.0-update-server_environment
```
Comment (JuMiSanAr) [inline oca_dependencies.txt]: Needs a rollback once the linked PR (#89) is merged

File: mail_environment/README.rst [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+.. image:: https://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+This module is part of the `OCA/server-env <https://github.com/OCA/server-env/tree/14.0/mail_environment>`_ project on GitHub.
```
Comment (santostelmo) [inline mail_environment/README.rst:141]:
```suggestion
This module is part of the `OCA/server-env <https://github.com/OCA/server-env/tree/15.0/mail_environment>`_ project on GitHub.
```
Replace reference to 14.0 by 15.0

**Review by simahawk — APPROVED** [review]

**Review by santostelmo — APPROVED** [review]

**Pull request merged**